<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ventana_imagenes</class>
 <widget class="QDialog" name="Ventana_imagenes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QPushButton" name="cerrar">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>580</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold 10pt &quot;Arial&quot;; 
color: #FFFFFF;
background-color: rgb(0, 0, 117); 
border-radius: 10px; </string>
   </property>
   <property name="text">
    <string>CERRAR SESIÓN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="info_im">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>30</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold 10pt &quot;Arial&quot;; 
color: #FFFFFF;
background-color: rgb(0, 0, 117); 
border-radius: 10px; </string>
   </property>
   <property name="text">
    <string>Ver información 
del paciente</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>520</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Malgun Gothic&quot;;
color: black;
background-color:rgb(169, 228, 255) ; 
border: 2px solid black;
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;VISTA SAGITAL&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>90</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Malgun Gothic&quot;;
color: black;
background-color:rgba(169, 228, 255,0.8) ; 
border: 2px solid black;
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;VISTA AXIAL&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>300</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Malgun Gothic&quot;;
color: black;
background-color:rgba(169, 228, 255,0.8) ; 
border: 2px solid black;
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;VISTA CORONAL&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="eje_axial" native="true">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>221</width>
     <height>191</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(100, 100, 100);
border: 2px solid rgb(0, 0,0)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="volver">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>580</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold 10pt &quot;Arial&quot;; 
color: #FFFFFF;
background-color: rgb(0, 0, 117); 
border-radius: 10px; </string>
   </property>
   <property name="text">
    <string>VOLVER</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>851</width>
     <height>651</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>foto analisis.jpeg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="valor_maximo">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>100</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt bold &quot;Consolas&quot;;
color: black;
background-color:rgba(255, 255, 255,0.2) ; 
</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="valor_actual">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>100</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt bold &quot;Consolas&quot;;
color: black;
background-color:rgba(255, 255, 255,0.2) ; 
 
</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>100</y>
     <width>21</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt bold &quot;Consolas&quot;;
color: black;
background-color:rgba(255, 255, 255,0.2) ; </string>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QSlider" name="slider">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>41</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black;
background-color:rgba(169, 228, 255,0.8) ; 
border: 2px solid black</string>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Vertical</enum>
   </property>
  </widget>
  <widget class="QWidget" name="eje_coronal" native="true">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>221</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(100, 100, 100);
border: 2px solid rgb(0, 0,0)</string>
   </property>
  </widget>
  <widget class="QWidget" name="eje_sagital" native="true">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>440</y>
     <width>221</width>
     <height>191</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(100, 100, 100);
border: 2px solid rgb(0, 0,0)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>90</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color:rgba(169, 228, 255,0.8) ; 
border: 2px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_5</zorder>
  <zorder>cerrar</zorder>
  <zorder>info_im</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>eje_axial</zorder>
  <zorder>volver</zorder>
  <zorder>slider</zorder>
  <zorder>eje_coronal</zorder>
  <zorder>eje_sagital</zorder>
  <zorder>label_6</zorder>
  <zorder>valor_actual</zorder>
  <zorder>label</zorder>
  <zorder>valor_maximo</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
